packages/dclP4DTensorFlow.dproj

```diff
@@ -7,12 +7,17 @@
         <MainSource>dclP4DTensorFlow.dpk</MainSource>
         <Platform Condition="'$(Platform)'==''">Win32</Platform>
         <ProjectGuid>{CCFDA41F-5004-4E54-997D-5967D19564F7}</ProjectGuid>
-        <ProjectVersion>19.4</ProjectVersion>
```

Comment:
What is the purpose of these changes? Should we keep it?

Reply:
May have been CE making numbering changes although I did have to alter a library version in another PR

Reply:
We can avoid these .dproj changes in existing packages.

Reply:
Dunno, may affect backward compatibility?

Not sure but thern again not had any problems with them myelf

Reply:
We are using Win32 Release as the default target platform and build configuration. It has changed in the .dproj files you've shared.

Reply:
I'm merging it up and doing the adjustments myself. You did it great, thanks! 😄